backend/src/main/java/gov/cdc/usds/simplereport/api/model/UpdateFacilityInput.java

```diff
@@ -0,0 +1,23 @@
+package gov.cdc.usds.simplereport.api.model;
+
+import java.util.List;
+import java.util.UUID;
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class UpdateFacilityInput {
+  private UUID facilityId;
+  private String facilityName;
+  private String cliaNumber;
+  private String street;
+  private String streetTwo;
+  private String city;
+  private String state;
+  private String zipCode;
+  private String phone;
+  private String email;
+  private ProviderInput orderingProvider;
+  private List<UUID> deviceIds;
```

Comment:
Sonar is complaining about this being a codesmell of duplicated code from `AddFacilityInput.java`. I believe we can make this easier to maintain and fix that code smell if we just `extend AddFacilityInput` and replace the standard `@Builder` with [`@SuperBuilder`](https://projectlombok.org/features/experimental/SuperBuilder), which creates a builder with the superclass!

Reply:
Looked into this more after @johanna-skylight made the change and it seems that `@SuperBuilder` creates a constructor that gets called when creating that object rather than using the builder which throws a `BindingException`. I am not aware of any way of using anything form lombok to fix this issue.